src/server/server/handlers.py

```diff
@@ -1,69 +1,153 @@
 import webapp2
+from webapp2_extras import sessions
 import dal  # pylint: disable=relative-import
 import jinjaenvironment  # pylint: disable=relative-import
 import frontendstructs as struct  # pylint: disable=relative-import
+import frontendhelpers as helpers  # pylint: disable=relative-import
 
 
 class Link(object):
 
-    def __init__(self, link, text):
-        self.link = link
+    def __init__(self, key, text):
+        self.key = key
         self.text = text
 
 
-class Feature(object):
+# How to use sessions :
+# http://stackoverflow.com/questions/14078054/gae-webapp2-session-the-correct-process-of-creating-and-checking-sessions
+class BaseHandler(webapp2.RequestHandler):
```

Comment:
I found the template in a more official doc : http://webapp-improved.appspot.com/api/webapp2_extras/sessions.html

Maybe this is more correct and likely to be updated if needed ?